### ISSUE TITLE
Please write a simple user guide on the tool

### ISSUE THREAD

**Issue by john-tau** [mysiar/api-platform-doc-parsing-check#1]
Hi it would be great if you could write a little bit more on how to use this tool in the READ.ME. Maybe I am being to rookie to ask such question.

**Comment by mysiar** [comment]
@john-tau is this gonna help ?
https://www.npmjs.com/package/@api-platform/api-platform-doc-parsing-check/tutorial

**Comment by john-tau** [comment]
@mysiar 

I did the below an worked you might consider update the "README.md" if you think is worth it. Thank for the prompt reply. 

1. yarn global add @api-platform/api-platform-doc-parsing-check 
The above wil add your utility in the global
2. How to use:
api-platform-doc-parsing-check  [api url in my test I used] as below
api-platform-doc-parsing-check  https://demo.api-platform.com  which gave me 

Checking documentation parsing for API @ https://demo.api-platform.com
[ 'books', 'reviews' ]
Number of endpoints parsed : 2
